SourceryRuntime/Sources/Method.swift

```diff
@@ -157,6 +157,11 @@ public final class MethodParameter: NSObject, SourceryModel, Typed, Annotated {
         return selectorName.hasPrefix("init(")
     }
 
+    /// Whether method is an deinitializer
+    public var isDeinitializer: Bool {
+        return selectorName.hasPrefix("deinit")
```

Comment:
Why not testing for equality here?
Maybe someone could create a `deinitMyObjects()` and I suppose this would report it as a _deinitializer_.